repos/system_upgrade/el7toel8/actors/cupsmigrate/actor.py

```diff
@@ -0,0 +1,339 @@
+from leapp.actors import Actor
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.common.reporting import report_generic
+from leapp.models import Report, Cupsmigrationmodel
+from leapp.tags import ApplicationsPhaseTag, IPUWorkflowTag
+
+import os
+from re import compile, findall, search, sub, MULTILINE
+
+"""
+CUPS configuration files
+"""
+CUPSD_CONF = '/etc/cups/cupsd.conf'
+CUPSFILES_CONF = '/etc/cups/cups-files.conf'
+
+"""
+Names for better navigation through change list
+"""
+
+INCLUDE = 0
+DIGEST = 1
+ENV = 2
+PRINTCAP = 3
+
+CERTKEY = 0
+
+KEY = 0
+NAMES = 1
+NEWNAME = 2
+SOURCE = 3
+TARGET = 4
+
+"""
+List of changed directives
+
+Scheme:
+Dictionary key # Directives # Input configuration file # Output configuration file
+
+Some changes happened only in one configuration file, so the last member of list is 'None'.
+"""
+cupsd_changes_list = [
+                      ['include', ['Include'], None, CUPSD_CONF, None],
+                      ['digest', ['Digest', 'BasicDigest'], 'Basic', CUPSD_CONF, None],
+                      ['env', ['SetEnv', 'PassEnv'], None, CUPSD_CONF, CUPSFILES_CONF],
+                      ['printcap', ['PrintcapFormat'], None, CUPSD_CONF, CUPSFILES_CONF]
+                    ]
+
+cupsfiles_changes_list = [
```

Comment:
ditto for using a dictionary or namedtuples.

Reply:
Using different approach now.